openapi-v2.yaml

```diff
@@ -2,12 +2,11 @@ openapi: 3.0.3
 info:
   title: dbt Cloud API v2
   version: 2.0.0
-  description: "# How to use this API\n\nThe dbt Cloud API is intended for enqueuing\
-    \ runs from a job, polling for run progress,\nand downloading artifacts after\
-    \ jobs have completed running. Operational endpoints around\ncreating, modifying,\
-    \ and deleting _objects_ in dbt Cloud are still in flux. These endpoints\nare\
-    \ largely undocumented in API v2.\n\nThe API docs are generated from an openapi\
-    \ spec defined in the\n[dbt-cloud-openapi-spec](https://github.com/dbt-labs/dbt-cloud-openapi-spec)\n\
+  description: "# How to use this API\n\nThe dbt Cloud API v2 contains endpoints for\
+    \ enqueuing runs from a job, polling for run progress,\nand downloading artifacts\
+    \ after jobs have completed running. For operational endpoints to\ncreate, modify,\
+    \ and delete _objects_ in dbt Cloud, consider using [API v3](/dbt-cloud/api-v3#/)\n\
```

Comment:
I'm pretty sure this link won't work. Can we update in dbt-cloud to be the full URL?